hummingbot/connector/exchange/ascend_ex/ascend_ex_auth.py

```diff
@@ -14,6 +16,33 @@ def __init__(self, api_key: str, secret_key: str):
         self.api_key = api_key
         self.secret_key = secret_key
 
+    async def rest_authenticate(self, request: RESTRequest) -> RESTRequest:
+        """
+        Adds the server time and the signature to the request, required for authenticated interactions. It also adds
+        the required parameter in the request header.
+        :param request: the request to be configured for authenticated interaction
+        """
+        # Generates auth headers
+        if request.is_auth_required:
```

Comment:
If this is executed it means `is_auth_required` is True in the request. No need to check again.